content/realtime/channels/channel-parameters/deltas.textile

```diff
@@ -0,0 +1,207 @@
+---
+title: Delta compression
+section: realtime
+index: 3
+jump_to:
+  Help with:
+    - Overview#overview
+    - Delta processing#delta-processing
+    - Using deltas#using-deltas
+    - Examples#examples
+---
+
+h2(#overview). Overview
+
+The @delta@ parameter enables delta compression. It is applied on the channel you are subscribing to, enabling delta mode. 
+
+Delta mode is a way for a client to subscribe to a channel so that message payloads sent contain only the difference (ie the delta) between the present message and the previous message on the channel.
+
+This is useful for channels that carry messages representing a series of updates to a particular object or document with a significant degree of similarily between successive messages. The client can apply the delta to the previous message to obtain the full payload. Using delta mode can signficantly reduce the encoded size of each message in the case that message payloads change by differences that are small relative to the size of the value. This reduction in size can reduce bandwidth costs, reduce transit latencies, and enable greater message throughput on a connection.
```

Comment:
@tomczoink do we have a process for requesting diagrams from the design team?

Reply:
Not actually sure these days, @remotekieran would you happen to know?

Reply:
If you need a diagram, we have a design request form: https://form.asana.com?hash=d0adca91644c74377be7ad0f3eef0033c5c681c3cc54bc1c8f5834d71c069811&id=1147945155091193

As much notice as possible is appreciated, especially considering the design team is part-time or part of the Experts Network @tomasmcm @MarkWoulfeAbly

Reply:
The quickest process for diagrams is you sketch one on paper, and then we ask Leonie to help.  No one is likely to design a diagram as useful as the person who is deep in the content and understands what needs visualising.  Can you sketch something on paper perhaps and I am sure Leonie could squeeze it in

Reply:
Thanks @paddybyers will help us draw something up, can I ask what the process for passing things to leonie is, and also who she is? I've not been intro'd.

Reply:
I think you meant me not @paddybyers @MarkWoulfeAbly :)

Anyway, go into the #team-design channel, and shout out to `@Leonie Wharton`.  She will happily help you out.  I am happy to do an intro to if you prefer, just ping me on Slack and I will do it

Reply:
Can I ask that you still fill out the design form with your request and rough sketches @MarkWoulfeAbly ?

The process we've defined for requesting design work is to fill out that form rather than ad hoc Slack messages that are harder for us to track. It also provides some semblance of a proper brief for Leonie to work from by providing enough info, context, and resources for her to have a solid first draft.

It doesn't need to slow the process down and doesn't mean you can't then ping Leonie in Slack with that brief, it just helps us maintain a proper influx and definition of work to be done. cc @mattheworiordan

Reply:
Form submitted, will add the diagram when it is done.

Reply:
Ah, sorry @remotekieran.  I was unaware of that process. My bad.